drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/BRLActionColumnPluginChildColumnUpdateTest.java

```diff
@@ -0,0 +1,172 @@
+                                                                                                 mock(DecisionTablePopoverUtils.class));
+
+        doReturn(EXTENDED_ENTRY).when(model).getTableFormat();
+        doReturn(model).when(presenter).getModel();
+        doReturn(mock(EventBus.class)).when(presenter).getEventBus();
+        wizard.init(presenter);
+        plugin.init(wizard);
+    }
+
+    public BRLActionColumn getBrlActionColumnWithTemplateValues() {
+        final BRLActionColumn brlActionColumn = new BRLActionColumn();
+
+        final ActionInsertFact actionInsertFact = new ActionInsertFact("Person");
+        actionInsertFact.setBoundName("$a");
+        final ActionFieldValue afv1 = new ActionFieldValue("name",
+                                                           "$default",
+                                                           DataType.TYPE_STRING);
+        afv1.setNature(FieldNatureType.TYPE_TEMPLATE);
+        actionInsertFact.addFieldValue(afv1);
+        final ActionFieldValue afv2 = new ActionFieldValue("age",
+                                                           "$default1",
+                                                           DataType.TYPE_STRING);
+        afv2.setNature(FieldNatureType.TYPE_TEMPLATE);
+        actionInsertFact.addFieldValue(afv2);
+        brlActionColumn.getDefinition().add(actionInsertFact);
+        brlActionColumn.getChildColumns().add(new BRLActionVariableColumn("$default", BRLActionVariableColumn.FIELD_VAR_NAME));
+        brlActionColumn.getChildColumns().add(new BRLActionVariableColumn("$default1", BRLActionVariableColumn.FIELD_VAR_NAME));
+        return brlActionColumn;
+    }
+
+    public BRLActionColumn getBrlActionColumnWithNoTemplateValues() {
+        final BRLActionColumn brlActionColumn = new BRLActionColumn();
+
+        final ActionInsertFact actionInsertFact = new ActionInsertFact("Person");
+        actionInsertFact.setBoundName("$a");
+        final ActionFieldValue afv1 = new ActionFieldValue("name",
+                                                           "Toni",
+                                                           DataType.TYPE_STRING);
+        afv1.setNature(FieldNatureType.TYPE_LITERAL);
+        actionInsertFact.addFieldValue(afv1);
+        final ActionFieldValue afv2 = new ActionFieldValue("age",
+                                                           "12",
+                                                           DataType.TYPE_STRING);
+        afv2.setNature(FieldNatureType.TYPE_LITERAL);
+        actionInsertFact.addFieldValue(afv2);
+        brlActionColumn.getDefinition().add(actionInsertFact);
+        final BRLActionVariableColumn variableColumn = new BRLActionVariableColumn("", BRLActionVariableColumn.FIELD_VAR_NAME);
+        variableColumn.setDefaultValue(new DTCellValue52("test"));
+        brlActionColumn.getChildColumns().add(variableColumn);
+        return brlActionColumn;
+    }
```

Comment:
Can we make these methods private please?

Reply:
Sure. For some reason I managed to make Idea set all the generated methods to public and not private and now I'm not sure how to reset it.